tags

```diff
@@ -0,0 +1,337 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
I think this file can be removed?!

Reply:
Thanks for catching that!  I'll add it to .gitignore.